SkyhookGame/Assets/Scripts/LevelUpSO/Institutions/HumanSectorLevelUpSO.cs

```diff
@@ -0,0 +1,13 @@
+﻿using UnityEngine;
+
+[CreateAssetMenu(fileName = "SO", menuName = "Instituion/Create Human Sector Level Up SO")]
+public class HumanSectorLevelUpSO : LevelUpSOBase
+{
+    public int minHumans;
+    public int maxHumans;
+
+    public int Evaluate(int currentLevel)
+    {
+        return Evaluate(currentLevel, 1, StaticVariables.MaxLevel, minHumans, maxHumans);
```

Comment:
Use base.Evaluate, because the names are conflicting

Reply:
Agreed, good point